pandas/core/aggregation.py

```diff
@@ -61,8 +60,8 @@ def normalize_keyword_aggregation(kwargs: dict) -> Tuple[dict, List[str], List[i
 
     Examples
     --------
-    >>> normalize_keyword_aggregation({'output': ('input', 'sum')})
-    ({'input': ['sum']}, ('output',), [('input', 'sum')])
+    >>> normalize_keyword_aggregation({"output": ("input", "sum")})
+    (defaultdict(<class 'list'>, {'input': ['sum']}), ('output',), array([0]))
```

Comment:
hmm, this output doesn't match the returns section/return type hint.

Reply:
@simonjayhawkins Fixed it in eceb678